### PR TITLE
pulpcore 3.14 settings

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by zjhuntin — APPROVED** [review]

File: releases/pulpcore/3.14/settings [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FULLVERSION='3.14'
+FULLGPGKEY='D052BF3E40E5A7E87F1208D99A1F620E6D05E64C'
```
Comment (ekohl) [inline releases/pulpcore/3.14/settings:2]: I don't see this release in gopass?

**Review by ekohl — COMMENTED** [review]
@zjhuntin when merging these PRs, please validate you can import the private key as well (and it matches). That way we're guaranteed that it's not just the author who can release.

File: releases/pulpcore/3.14/settings (hunk 1 above)
Comment (evgeni) [reply releases/pulpcore/3.14/settings:2]: try again ;)

Comment (ekohl) [reply releases/pulpcore/3.14/settings:2]: I still don't?

Comment (ekohl) [reply releases/pulpcore/3.14/settings:2]: Now I do :)

**Pull request merged**